client/src/components/AlumniCollegeCareerUpdateModal.js

```diff
@@ -0,0 +1,340 @@
+import React, { Component } from 'react';
+import {Button, Modal, Divider, Step, Segment, Grid, Label, Icon, Dropdown } from 'semantic-ui-react';
+import PooledSingleSelectDropdown from "./PooledSingleSelectDropdown"
+import { makeCall } from "../apis";
+import swal from "sweetalert";
+
+/*
+    props:
+    - modalOpen
+    - closeModal
+    - getInput
+*/
+export default class AlumniCollegeCareerUpdateModal extends Component {
+    constructor(props){
+        super(props)
+        // props are currently not being used to populate input fields
+        // this can be added as a future enhancement
+        this.state = {
+            // Job Title
+            existingJobTitleId: '',
+            existingJobTitleName: '',
+            newJobTitle: '',
+            // Company
+            existingCompanyId: '',
+            existingCompanyName: '',
+            newCompany: '',
+            // College
+            countryOptions: [],
+            country: '',
+            newCollege: '',
+            existingCollegeId: '',
+            existingCollegeName: '',
+            collegeOptions: [],
+            // form control
+            submitting: false
+
+        }
+        // job title
+        this.getJobTitleInput = this.getJobTitleInput.bind(this)
+        // company
+        this.getCompanyInput = this.getCompanyInput.bind(this)
+        // college
+        this.handleCountrySelection = this.handleCountrySelection.bind(this)
+        this.getCollegeInput = this.getCollegeInput.bind(this)
+        this.deleteCountry = this.deleteCountry.bind(this)
+        this.submit = this.submit.bind(this)
+        this.submitReady = this.submitReady.bind(this)
+        this.clearState = this.clearState.bind(this)
+    }
+
+    async componentWillMount() {
+        let result = await makeCall(null, '/drop/countries', 'get')
+        this.setState({countryOptions: result.options})
+    }
+
+    clearState() {
+        this.setState({
+            existingJobTitleId: '',
+            existingJobTitleName: '',
+            newJobTitle: '',
+            // Company
+            existingCompanyId: '',
+            existingCompanyName: '',
+            newCompany: '',
+            // College
+            country: '',
+            newCollege: '',
+            existingCollegeId: '',
+            existingCollegeName: '',
+            collegeOptions: [],
+        })
+    }
+
+    getCompanyInput(selection, isNew) {
```

Comment:
As a reminder, these handler methods are necessary to lift state out of pooled dropdowns